src/parse/asp/grammar.go

```diff
@@ -166,6 +166,8 @@ type IdentStatementAction struct {
 // An IdentExpr implements parts of an expression that begin with an identifier (i.e. anything
 // that might be a variable name).
 type IdentExpr struct {
+	Pos Position
```

Comment:
Actually, the `Pos` here is not going to be the same in some scenarios. For example, in the case of arguments or assignments. like so:
`options = ' '.join(['-o ' + option for option in optional]) if optional else ''`
The parent statement Pos would have been `line:1, column:1`, however, in this case, I will need to get the position of the `IdentExpr`, which is the `join()` call, since I need to know if the mouse hover is actually on the function call. I know it looks like these might be repeating, but they are actually getting different things for my use case.